src/pages/Dashboard.js

```diff
@@ -0,0 +1,24 @@
+import LaunchCard from '../components/LaunchCard';
+import useLaunchList from '../hooks/useLaunchList';
+import { CardList, Error, Loader } from './styledComponents';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
import { CardList, Error, Loader } from "./styledComponents";
```